code/index.html

```diff
@@ -1,22 +1,133 @@
 <!DOCTYPE html>
 <html>
   <head>
-    <meta charset="utf-8" />
-    <title>Project 2: The News Site</title>
+    <meta charset="utf-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1">
+    <title>SAFARISAFARIMAG</title>
     <link rel="stylesheet" href="./style.css" />
-    <!-- dont forget the meta tag for responsiveness -->
   </head>
-  <body>
+  <body>    
     <header>
-      <!-- Some kind of logo and navbar -->
+      <img class="logo" src="./images/logo2.png">
```

Comment:
Let's not forget about `alt` attribute

Reply:
Hi! Thank you for the responsen, I will take everything you said and have go through my code.

Did not know if it vendorprefixes was a good or bad idea. I actually tried to open the netlify-link on my phone yesterday (oneplus 6s) and it looked diffrent then in chromes phone-view. 

How do you style your responsive design? For example, do you change fontsize for diffrent mediaqueries? For you prefer to use vendorprefixes?

Thank you once again for the feedback / Mats

Reply:
Vendor prefixes was a very good idea :) Still, sometimes content of your website will looks different between browsers, it can be both because of styles and just native behaviour of how some HTML elements are displayed by browsers.

Yes, sometimes it's good idea to change font-size on different media queries. General rule is that text should be nicely visible for users.